adafruit_hid/keyboard.py

```diff
@@ -39,7 +35,7 @@ class Keyboard:
 
     # No more than _MAX_KEYPRESSES regular keys may be pressed at once.
 
-    def __init__(self, devices: Sequence[usb_hid.Device]) -> None:
+    def __init__(self, devices: list[usb_hid.Device]) -> None:
```

Comment:
@jepler Just a heads up that using built-ins as typing generics wasn't added until Python 3.9.

Reply:
what's the impact of this, unimportable on older python now?  what's our baseline python requirement, given that current raspberry pi os is on 3.9 by now?

Reply:
@makermelissa did I get that right?